ui/js/util/getThemes.js

```diff
@@ -0,0 +1,20 @@
+const { readdirSync } = require("fs");
```

Comment:
One problem that would occur from throwing it into a selector without that selector reading from the state is that the selector will be memoized so files would be read only once. Doesn't seem like a big concern though unless people are adding themes on the fly? I would have probably gone with action/reducer/selector. Modifying the build progress also seems fine to me.